### PR TITLE
on hover play for audio instead of click

### PR DESCRIPTION
**Pull request by mritunjayz**

### CODE REVIEW

**Comment by mritunjayz** [comment]
#2187 #2198

File: web/src/components/layout/user-menu.tsx [hunk 1]
```diff
@@ -38,23 +40,31 @@ export default function UserMenu() {
...
             trackVoiceAvatar('self-listen', locale);
             if (account.avatar_clip_url !== null && !showAnimation) {
-              const audio = new Audio(account.avatar_clip_url);
-              audio.play();
+              audioRef.current.src = account.avatar_clip_url;
+
+              audioRef.current.play();
               setShowAnimation(!showAnimation);
-              audio.onended = () => {
+              audioRef.current.onended = () => {
```
Comment (Gregoor) [inline web/src/components/layout/user-menu.tsx]: Why not attach these event listeners in the JSX?

File: web/src/components/layout/user-menu.tsx [hunk 2]
```diff
@@ -38,23 +40,31 @@ export default function UserMenu() {
         className={'user-menu ' + (showMenu ? 'active' : '')}
         onMouseEnter={() => setShowMenu(true)}
         onMouseLeave={() => setShowMenu(false)}>
+        <audio preload="auto" ref={audioRef} />
         <button
           className="toggle"
           title="click to play avatar"
-          onClick={() => {
```
Comment (Gregoor) [inline web/src/components/layout/user-menu.tsx]: For mobile we'd still need tap to listen, so I think we'd wanna have both mouseEnter and click

File: web/src/components/layout/user-menu.tsx (hunk 1 above)
Comment (mritunjayz) [reply web/src/components/layout/user-menu.tsx]: yeah, we can

**Pull request merged**